app/styles/component/_recommended-video-lazy.scss

```diff
@@ -170,3 +170,40 @@ $recommended-video-player-width: 200px;
 		display: none;
 	}
 }
+
+.is-click-to-play {
+	display: flex;
+	flex-wrap: wrap;
+	height: 100px;
+	width: 100%;
+
+	header {
+		width: 100%;
+	}
+
+	> div:nth-child(2) {
+		height: 52.3px;
```

Comment:
are decimals necessary?

Reply:
I've straight up copied it from zeppelin :)

Reply:
then round it to 52px